ui/components/app/add-network/index.scss

```
@@ -21,6 +24,12 @@
     cursor: pointer;
   }
 
+  &__add-button {
+    color: var(--primary-1);
```

Unfortunately we have some issues with the order of our stylesheets so you need to use more specificity here. Now that we have updated it to using the Button component
```suggestion
  &__add-button.button {
    color: var(--color-primary-default);
```

---

Done